packages/core/src/utils.js

```diff
@@ -108,11 +108,7 @@ export function getWidget(schema, widget, registeredWidgets = {}) {
     return Widget.MergedWidget;
   }
 
-  if (
-    typeof widget === "function" ||
-    ReactIs.isForwardRef(React.createElement(widget)) ||
-    ReactIs.isMemo(widget)
-  ) {
+  if (typeof widget !== "string" && ReactIs.isValidElementType(widget)) {
     return mergeOptions(widget);
```

Comment:
Since our old tests didn't catch this issue, would it be possible to update them so that we don't regress again?

Reply:
Yes, I'll do that soon. Thanks for the reminder